backend/globaleaks/tests/handlers/test_notification.py

```diff
@@ -18,8 +18,11 @@ class TestNotificationInstance(helpers.TestHandlerWithPopulatedDB):
     @inlineCallbacks
     def test_get(self):
         handler = self.request(role='admin')
+
         yield handler.get()
+
         self.assertEqual(self.responses[0]['server'], 'demo.globaleaks.org')
+        self.assertTrue('exception_email_pgp_key_info' not in self.responses[0])
```

Comment:
i'm more for implementing a retesting that based on data definitions in requests.py will check presence of all keys expected to be in the REST interface and would check that no additional keys would be there.

we can have a ```self.ValidateDataStructure(dictionary, dictionaryDef)``` were to use a dictionaryDef from request.py and eventually arrive to have soomething aotomagic from models.py where to have filters depending on users role but this part only in the future future! :)